tests/src/AppBundle/Book.php

```diff
@@ -23,7 +23,7 @@ public function getTitle()
     /**
      * @param string $title
      */
-    public function setTitle($title)
+    public function setTitle(string $title)
```

Comment:
No, we still support php5.

Reply:
yeah I know, it's just my path to resolve all of this ;) things will become green soon.